docs/Introduction.md

```diff
@@ -28,10 +28,22 @@ The latest version runs on macOS 10.10 or later. [Previous versions](https://qsa
 
 ## Installing Quicksilver
 
+From the Disc Image (.dmg) file:
+
   1. [Download Quicksilver](https://qsapp.com/download.php).
   2. Open the `.dmg` file and drag Quicksilver to your Applications folder.
   3. Unmount the Quicksilver volume and delete the `.dmg` file.
 
+Using Homebrew:
+
+[Homebrew is a package manager for OS X](https://brew.sh) for installs, updates and uninstalls of most OS X software:
+
+Open Terminal, type `brew install --cask quicksilver`. You can `uninstall` just as easily. 
+
+## First Launch
+
+Due to the way Quicksilver is built, OS X raises a [code signing warning (open issue)](https://github.com/quicksilver/Quicksilver/issues/2576))—the quickest way to get around this is to <kbd>ctrl-</kbd> or <kbd>right-click</kbd> the Quicksilver app in Applications, choose "open" from the contextual menu, *then* choose the *extra "open" button* that now appears in the warning dialog box. This only needs doing once, Quicksilver will then launch in future without further issue.
```

Comment:
Hi. 

Maybe the `OS X` reference should be changed to `macOS` ??? 

https://en.wikipedia.org/wiki/MacOS

Reply:
Agreed . Old habits! macOS it is, not OS X.
```suggestion
Due to the way Quicksilver is built, macOS raises a [code signing warning (open issue)](https://github.com/quicksilver/Quicksilver/issues/2576))—the quickest way to get around this is to <kbd>ctrl-</kbd> or <kbd>right-click</kbd> the Quicksilver app in Applications, choose "open" from the contextual menu, *then* choose the *extra "open" button* that now appears in the warning dialog box. This only needs doing once, Quicksilver will then launch in future without further issue.
```